lib/src/executables/sort_intl.dart

```diff
@@ -0,0 +1,156 @@
+// Copyright 2023 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:io';
+import 'package:analyzer/dart/analysis/utilities.dart';
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:args/args.dart';
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:logging/logging.dart';
+import 'package:over_react_codemod/src/intl_suggestors/intl_messages.dart';
+
+import 'package:over_react_codemod/src/util/package_util.dart';
+import 'package:path/path.dart' as p;
+import '../util.dart';
+import 'intl_message_migration.dart';
+
+final _log = Logger('orcm.intl_message_migration');
+
+const _noMigrate = 'no-migrate';
+
+final FileSystem fs = const LocalFileSystem();
+
+final parser = ArgParser()
+  ..addFlag(
+    'help',
+    abbr: 'h',
+    negatable: false,
+    help: 'Prints this help output.',
+  )
+  ..addFlag(_noMigrate,
+      negatable: false,
+      defaultsTo: false,
+      help:
+      'Does not run any migrators, overriding any --migrate flags. Can still be used with --prune-unused, and '
+          'will force the messages file to be sorted and rewritten'
+  );
+
+late ArgResults parsedArgs;
+
+void main(List<String> args) async {
+  parsedArgs = parser.parse(args);
+  if (parsedArgs['help'] as bool) {
+    printUsage();
+    return;
+  }
+
+  // If we have specified paths on the command line, limit our processing to
+  // those, and make sure they're absolute.
+  var basicDartPaths = parsedArgs.rest.isEmpty ? ['lib'] : parsedArgs.rest;
```

Comment:
We don't care about paths other than for the _intl.dart files. So this could probably be replaced with something like find all '*_intl.dart' files. But the IntlMessage class already knows how to look for those, so I'd say just don't do this at all, don't compute the files.